src/Integration/Suppression/SuppressedIssueMatcher.cs

```diff
@@ -51,13 +51,12 @@ public bool SuppressionExists(IFilterableIssue issue)
 
             // File-level issues never match non-file-level issues.
 
-            // Retrieve all issues relating to this file (file level or precise location) or project (for module level issues)
-            var serverIssues = issuesProvider.GetSuppressedIssues(issue.ProjectGuid, issue.FilePath);
+            var serverIssues = serverIssuesStore.Get();
 
             // Try to find an issue with the same ID and either the same line number or some line hash
-            bool matchFound = serverIssues.Any(s => IsMatch(issue, s));
+            bool foundIssueIsSuppressed = serverIssues.Any(s => IsMatch(issue, s) && s.IsResolved);
```

Comment:
Not a huge fan of the name but I felt like it needed to be changed

Reply:
Alternatives: `isSuppressed`, or `matchesSuppressedIssue`.

Perf: checking `s.IsResolved` will be fast because it is just checking a property rather than calling a method, so it would be better to put that check first.